.github/CHANGELOG.md

```diff
@@ -21,6 +28,11 @@
   as a class attribute.
   [(#46)](https://github.com/XanaduAI/blackbird/pull/46)
 
+* Remove p-type parameters from `prog.parameters` since they're only needed there internally and
+  are otherwise stored in `prog.variables`. This also changes TDM programs to not be templates
+  (`prog.is_template` now returns `False`).
```

Comment:
Just out of curiosity - is there a consequence of TDM programs not being templates any more?

Reply:
I don't think so, no. As far as I know this is just a consequence of how we decided to implement these p-type parameters internally. It only blocks the use of the `match_template` function for all TDM programs (this fixes that).